tailwind.config.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
         green: 'var(--color-green)',
         yellow: 'var(--color-yellow)',
         white: 'var(--color-white)',
-        lightgreen: 'var(--color-lightgreen)'
+        lightgreen: 'var(--color-lightgreen)',
+        black: 'var(--color-black)'
```

Comment:
We added oebblack as tailwind variable in the latest merges, because we will most likely need a "real" black and the almost black we are using for texts etc...